phoebe/backend/universe.py

```diff
@@ -2432,7 +2435,7 @@ def is_convex(self):
     def needs_recompute_instantaneous(self):
         # recompute instantaneous for asynchronous spots, even if meshing
         # doesn't need to be recomputed
-        return self.needs_remesh or (not self.is_single and len(self.features) and self.F != 1)
+        return self.needs_remesh or (len(self.features) and self.F != 1.0) or self.is_single
```

Comment:
won't this force recomputing whenever self.is_single is True? Why is this necessary?

Reply:
we definitely want to only do this when absolutely necessary